### ISSUE TITLE
[Linux] keyman stops working when textline reaches some length limit

### ISSUE THREAD

**Issue by nikita-moor** [keymanapp/keyman#2002]
**Describe the bug**

LibreOffice Writer and gedit (including its fork mousepad) stop recognizing character sequences, such as `-a > ā`, when the length of the text line is more than 60 symbols:

```
āāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāāā-a-a-a-a-a-a-a
ēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēēē
```

Typing "-e" in the second line, you can add one more symbol "ē", but following typing will not transform into the compound symbol.

**To Reproduce**

1. Start `gedit`
2. Create an empty document (File|New)
3. Star typing "-a"
4. When the line reaches length of 60 symbols "ā", you will see the bug.

---

**Keyman for Linux:**
 - OS: Manjaro (Arch Linux based)
 - Keyman version: 11.0.122
 - Target Application: LibreOffice, gedit
 
---
 
**Keyboard**
 - Keyboard name: EuroLatin (SIL)
 - Keyboard version: 1.8.1

---

**Notes**

Notepadqq is not affected, so it may be a GTK-related bug.

**Comment by mcdurdin** [comment]
We don't currently have anyone able to work on the Linux project; we'll take a look at this in 13.0. 

@glasseyes if you have some time, perhaps you'd like to take a look?

**Comment by darcywong00** [comment]
This will be fixed in #2607

**Comment by nikita-moor** [comment]
Confirm, the problem was solved. Thank you!
